src/app/components/HelpFAQ.tsx

```
@@ -375,19 +426,21 @@ export function HelpFAQ() {
-              something feels off, we'll make it right.
+              for support, questions, or thoughts you'd like to share. We read
+              every message, and if something feels off, we'll make it right.
             </p>
             <button
               className="px-8 py-4 border text-black font-din-arabic tracking-wide hover:bg-black hover:text-white transition-all duration-300 shadow-sm hover:shadow-md"
               style={{ borderColor: "#D8D2C7" }}
-              onClick={() => (window.location.href = "mailto:hello@jardinbotanica.com")}
+              onClick={() =>
+                (window.location.href = "mailto:hello@jardinbotanica.com")
+              }
             >
```

Biome (2.4.15)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)